### PR TITLE
Add pitch to FlxSound demo

### PR DESCRIPTION
**Pull request by ACrazyTown**
Adds a numeric stepper for the pitch property on every target except flash

### CODE REVIEW

File: Features/FlxSound/source/MenuState.hx [hunk 1]
```diff
@@ -22,6 +22,9 @@ class MenuState extends FlxUIState
 {
 	var loop_music:Bool = false;
 	var music_volume:Float = 0.5;
+	#if !flash
```
Comment (Geokureli) [inline Features/FlxSound/source/MenuState.hx]: All uses of `#if !flash` should be `#if FLX_PITCH` and `!flash` should be `FLX_NO_PITCH`

Comment (ACrazyTown) [reply Features/FlxSound/source/MenuState.hx]: i completely forgot about the defines, fixed

**Comment by ACrazyTown** [comment]
btw unrelated but I've noticed that HTML5 builds seem to include both mp3 and ogg files, despite the code [only using mp3 on flash](https://github.com/HaxeFlixel/flixel-demos/blob/dev/Features/FlxSound/source/MenuState.hx#L92-L97). is this an oversight?

**Comment by Geokureli** [comment]
> btw unrelated but I've noticed that HTML5 builds seem to include both mp3 and ogg files, despite the code [only using mp3 on flash](https://github.com/HaxeFlixel/flixel-demos/blob/dev/Features/FlxSound/source/MenuState.hx#L92-L97). is this an oversight?

you mean "despite the code only using mp3 on *html*"?

**Comment by Geokureli** [comment]
> btw unrelated but I've noticed that HTML5 builds seem to include both mp3 and ogg files, despite the code [only using mp3 on flash](https://github.com/HaxeFlixel/flixel-demos/blob/dev/Features/FlxSound/source/MenuState.hx#L92-L97). is this an oversight?

does ogg work on most browsers?
Edit: Yes. feel free to fix that here or in another PR

**Comment by ACrazyTown** [comment]
should be good to go now i think

**Comment by Geokureli** [comment]
Thanks

**Pull request merged**